### DISCUSSION TITLE
打電動會變聰明？

### DISCUSSION THREAD

**Discussion by AngusGuei** [1101-GS3073-C/Python-Community-GS3073-C#410] [General]
每次看到這種新聞都不知道是不是真的
但這篇據說是 ＡＢＣＤ發的 ，可信度偏高

https://technews.tw/2022/05/30/screen-video-games-smarter/

**Comment by lx3211** [comment]
但是我總感覺打LOL只會更容易暴躁而已XD

**Comment by 32844583** [comment]
打電動可能真的某方面會變聰明，前提是不沉癮偶爾打，然後選對的遊戲也是一大關鍵，像樓上可能就是選錯遊戲

**Comment by brian431** [comment]
確實有許多遊戲都會需要大量地思考，不過相反的也有很多遊戲是很無腦的，像這種的應該就對提升智力沒什麼幫助。

**Comment by NcuJerryKao** [comment]
玩遊戲 不乏戰略、技術、鬥智鬥勇 
兩者實力相當廝殺 當智取
自然會使你頭腦變靈活

**Comment by Aerofe** [comment]
可能要看每個人玩遊戲的目的吧，有些人只是想放鬆就可能不太會想要動腦。有些人會去思考戰略等⋯，就真的有可能會變聰明

**Comment by BingRong1217** [comment]
但同時也會有負面的影響

**Comment by Yilin20021016** [comment]
有不少電動可以訓練臨場反應 但是不包括手機遊戲

**Comment by HsuChen88** [comment]
有些遊戲可以防止老化，無腦的遊戲也可以放鬆心情，心態正確玩什麼遊戲都有幫助。

**Comment by skyling0299** [comment]
這邊推好遊戲celeste 優美風景多變玩法單一目標，讓你打發時間之餘也放鬆心靈學習直面挑戰!

**Reply by RogelioKG** [reply]
提到跑酷一律推薦空洞騎士的苦痛之路🥴

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by awehao** [comment]
我覺得我快變成傻子了了

**Comment by RogelioKG** [comment]
單純分享一下我在魂系遊戲的經驗：
- `空洞騎士`：300小時
- `黑魂3`：200小時 白金
- `隻狼`：90小時 白金
- `黑魂1`：50小時
- `法環`：想玩但是這學期很忙
很多人說魂系也可以訓練反應力，
所以說我的反應力有變好嗎？
有，真的有變好，至少遊戲層面來說。
從以前被朋友笑是手殘黨，到現在讀招和喝水一樣自然。
我覺得考驗反應力的遊戲就是去不斷強化這方面的神經連結，
久而久之，你就能對突發事件做出快速反應，甚至是肌肉記憶的反應，
所以說，現實生活中的我，也有收穫到反應力變好的紅利嗎？
答案是：並沒有

**Comment by Xuewen628** [comment]
我覺得重點是動腦吧 玩遊戲通常被當成浪費時間的事 但也有人從中學到很多

**Comment by Raywei0214** [comment]
還好我都打音遊，除了訓練反應力還可以訓練手指肌肉以及靈活度，非常推薦各位 プロセカ

**Comment by larry40305** [comment]
很多遊戲需要分析數值考慮戰術，應該多少可以幫助思考。

**Comment by jcpan821** [comment]
我覺得還是要看遊戲類型，像有些益智遊戲的確可以訓練邏輯思考

**Comment by boson13579** [comment]
我覺得有好有壞耶，好處是增加學童的反應力和團隊合作能力，但也有另一項研究表示學童可能因此變得更暴力。

**Comment by 710388** [comment]
透過電動孩子將經驗到很多本來不會接觸的情境。當知識面廣了，經驗增加，能力也會變強。
